include/zephyr/bluetooth/hci_vs.h

```diff
@@ -234,8 +232,10 @@ struct bt_hci_vs_fata_error_cpu_data_cortex_m {
 	uint32_t a4;
 	uint32_t ip;
 	uint32_t lr;
+	uint32_t pc;
```

Comment:
Does this count as a public API change? 

If anyone was using this VS command before, something may change. Given that this is just an addition, it should not be harmful I think